tests/producer/circle/test_core.py

```diff
@@ -3,9 +3,20 @@
 from chroma_feedback.producer.circle.core import fetch
 
 
+def test_fetch_only_mine() -> None:
```

Comment:
Please rename to `test_fetch_slug_mine_only` and change order after `test_fetch_slug`